lib/shared/addon/components/schema/input-boolean/component.js

```diff
@@ -1,14 +1,17 @@
 import Component from '@ember/component';
+import { next } from '@ember/runloop';
 import layout from './template';
 
 export default Component.extend({
   layout,
 
   didReceiveAttrs() {
-    if ( this.get('value') === 'false' ) {
-      this.set('value', false);
-    } else if ( this.get('value') === 'true' ) {
-      this.set('value', true);
-    }
+    next(() => {
+      if ( this.get('value') === 'false' ) {
```

Comment:
```
if (get(this, 'value)){
...
} else {
...
}
```

Reply:
Agree with true being first but they are string `true`/`false` so just "if (get(this, 'value)" doesn't work.